Packs/Venafi/Integrations/VenafiV2/VenafiV2.py

```diff
@@ -251,72 +245,50 @@ def get_certificate_details_command(client: Client, args: dict[str, Any]) -> Com
         client (Client): A Venafi client.
         args (dict): The arguments for the command passed to the request.
     Returns:
-        A CommandResult object with an outputs, raw_response and readable table, in case of a successful action.
+        A CommandResult object with an outputs, raw response and readable table, in case of a successful action.
     """
 
-    outputs: dict[str, Any] = {}
-    guid = args.get('guid', '')
-    response = client._get_certificate_details(guid)
-    if response:
-        outputs = response
-
-    human_readable = []
-    certificate_details = get_human_readable_object(outputs)
-    human_readable.append(certificate_details)
-
-    markdown_table = tableToMarkdown('Venafi certificate details', human_readable)
+    guid: str = args.get('guid', '')
+    response = client.get_certificate_details(guid)
+    response['ID'] = response.get('Guid', '').strip('{}')  # add ID to response for backward compatible with V1
+    del response['Guid']  # remove redundant entry since ID entry is the same.
+    readable_certificate_details = response.copy()
+    if not readable_certificate_details['ID']:
+        entries = []
+    else:
+        entries = [readable_certificate_details]
+    markdown_table = tableToMarkdown('Venafi certificate details', entries,
+                                     headers=['CreatedOn', 'DN', 'Name', 'ParentDn', 'SchemaClass', 'ID'])
 
     return CommandResults(
         outputs_prefix=CONTEXT_OUTPUT_BASE_PATH,
-        outputs=outputs,
-        raw_response=response,
-        readable_output=markdown_table
+        outputs=response,
+        raw_response=response if response else {},
+        readable_output=markdown_table,
+        outputs_key_field='ID'
     )
 
 
 ''' HELPER FUNCTIONS '''
 
 
-def get_human_readable_object(certificate: dict[str, Any]) -> Dict[str, Any]:
+def edit_response(certificates: list) -> list:
```

Comment:
If we have this method, we not running it at first and then use the output of it for the human-readable instead of doing it also for the human readable as done in the methods.